tests/integration/login.test.ts

```diff
@@ -0,0 +1,31 @@
+import { SkynetClient } from "skynet-js";
+
+import { randomUnicodeString } from "../utils";
+import { generatePhrase, phraseToSeed } from "../../src/seed";
+import { login, register } from "../../src/login";
+
+// TODO: Remove hard-coded URL.
+const portalUrl = "https://siasky.xyz";
+const client = new SkynetClient(portalUrl);
+const phrase = generatePhrase();
+const seed = phraseToSeed(phrase);
+const email = `${randomUnicodeString(20)}@bar.com`;
+
+describe("Integration tests for registration and login", () => {
+  it("should register a new user", async () => {
+    const jwt = await register(client, seed, email);
+
+    expect(jwt).not.toEqual("");
+    // We don't know the exact length.
+    expect(jwt.length).toBeGreaterThan(1000);
```

Comment:
Can we parse it and check that the email is correct? The length check doesn't verify the validity of the JWT.

Also, I've been meaning to reduce the size of the JWT in order to speed up requests on slow networks and this check `len > 1000` will not work well with that.

Reply:
How do I do that?

Reply:
Okay, I figured out how to parse it, but I don't see a email in there.

```
{
  exp: 1641143455,
  iat: 1638551455,
  iss: 'https://siasky.xyz',
  session: { active: true, identity: { traits: [Object] } },
  sub: '9ee640d03ee8460a813fda01a2e5ad0b'
}
```

Reply:
You'll have to dig a little deeper:
```json
{
    "exp": 1641379687,
    "iat": 1638787687,
    "iss": "https://",
    "session": {
        "active": true,
        "identity": {
            "traits": {
                "email": "testHandlerUserPOST@siasky.net"
            }
        }
    },
    "sub": "108eeca78d9a4135b508d52adc950d46"
}```

This structure comes from the ORY stack, in case you are wondering why we do it like that.